even-digits-/even-digits.go

```diff
@@ -0,0 +1,21 @@
+package main
+
+import ( 
+	"fmt"
+)
+
+func main() {
+	fmt.Println("The program determines if all digits in the integer are even.")
+	fmt.Println("Enter the number:")
+	var a int //-28; 379; 2806; 0; 33
```

Comment:
It's great that you provided the tests! However, instead of providing tests inline (as program comments), I suggest you to specify them before or after the program, e.g.

```
// Input: -28
// Output: All digits in -28 are even
//
// Input: 379
// Output: Not all digits in 379 are even
````

The tests are more readable in this way.

It applies also for other programs.